xds/internal/xdsclient/xdslbregistry/converter/converter.go

```diff
@@ -177,6 +181,29 @@ func convertWeightedRoundRobinProtoToServiceConfig(rawProto []byte, _ int) (json
 	return makeBalancerConfigJSON(weightedroundrobin.Name, lbCfgJSON), nil
 }
 
+func convertLeastRequestProtoToServiceConfig(rawProto []byte, _ int) (json.RawMessage, error) {
+	if !envconfig.LeastRequestLB {
+		return nil, nil
+	}
+	lrProto := &v3leastrequestpb.LeastRequest{}
+	if err := proto.Unmarshal(rawProto, lrProto); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal resource: %v", err)
+	}
+	// "The configuration for the Least Request LB policy is the
+	// least_request_lb_config field. The field is optional; if not present,
+	// defaults will be assumed for all of its values." - A48
+	var choiceCount uint32 = defaultLeastRequestChoiceCount
```

Comment:
Not asking you to change, but just FYI `choiceCount := uint32(defaultLeastRequestChoiceCount)` is shorter.

Reply:
Switched. I feel like throughout the codebase and my years on the team, x := y vs. var x type = y has always been preferred for local vars declaration to a non zero value. Noted for future.